gitops/flux-region2.yaml

```diff
@@ -0,0 +1,184 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app.kubernetes.io/name: flux
+  name: flux
+  namespace: cluster-baseline-settings
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: flux
+  labels:
+    app.kubernetes.io/name: flux
+rules:
+  - apiGroups: ['*']
+    resources: ['*']
+    verbs: ['*']
+  - nonResourceURLs: ['*']
+    verbs: ['*']
```

Comment:
Shouldn't most of the flux resources be in the bases isntead of dupliated?  The only thing that I would think would not be in the bases would be either the patch to override merge the `git-path` args.

Reply:
I'm totally up for that; I figured something like that would be possible (started with brute-force approach to get it working). Please let me take care of this as a bug fix in the next PR.

Reply:
change of plans, this change is better to be tackled from here. I'm taking care of this now.